workflow/cron/operator.go

```diff
@@ -321,3 +325,14 @@ func (woc *cronWfOperationCtx) reportCronWorkflowError(conditionType v1alpha1.Co
 	})
 	woc.metrics.CronWorkflowSubmissionError()
 }
+
+func inferScheduledRuntime() time.Time {
+	// Infer scheduled runtime by getting current time and zeroing out current seconds and nanoseconds
+	// This works because the finest possible scheduled runtime is a minute
+	now := time.Now().UTC()
+	return time.Date(now.Year(), now.Month(), now.Day(), now.Hour(), now.Minute(), 0, 0, now.Location())
+}
+
+func getChildWorkflowName(cronWorkflowName string, scheduledRuntime time.Time) string {
+	return fmt.Sprintf("%s-%d", cronWorkflowName, scheduledRuntime.Unix())
```

Comment:
truncate time to 1 minute?

Reply:
I think this function should accept `scheduledRuntime` as given and not modify it

Reply:
do you truncate there?